docs/source/Container.rst

```diff
@@ -0,0 +1,271 @@
+   
+.. code-block:: 
+
+   mkdir /lustre/cache
+   mkdir /lustre/tmp
+   sudo su
+   export SINGULARITY_CACHEDIR=/lustre/cache
+   export SINGULARITY_TMPDIR=/lustre/tmp
+   exit
+
+.. note:: 
+   ``/lustre`` is a fast but non-persistent file system used on NOAA Cloud systems. To retain work completed in this directory, `tar the files <https://www.howtogeek.com/248780/how-to-compress-and-extract-files-using-the-tar-command-on-linux/>`__ and move them to the ``/contrib`` directory, which is much slower but persistent.
+
+.. _ContainerBuild:
+
+Build the Container
+======================
+
+Set a top-level directory location for Land DA work, and navigate to it. For example:
+
+.. code-block:: console 
+
+   export LANDDAROOT=/path/to/landda
+   [[ -d $LANDDAROOT ]] || mkdir -p $LANDDAROOT 
+   cd $LANDDAROOT
+
+where ``/path/to/landda`` is the path to this top-level directory (e.g., ``/Users/Joe.Schmoe/landda``). The second line will create the directory if it does not exist yet. 
+
+.. hint::
+   If a ``singularity: command not found`` error message appears in any of the following steps, try running: ``module load singularity``.
+
+NOAA RDHPCS Systems
+----------------------
+
+On many NOAA :term:`RDHPCS` systems, a container named ``ubuntu20.04-intel-landda.img`` has already been built, and users may access the container at the locations in :numref:`Table %s <PreBuiltContainers>`.
+
+.. _PreBuiltContainers:
+
+.. table:: Locations of Pre-Built Containers
+
+   +--------------+--------------------------------------------------------+
+   | Machine      | File location                                          |
+   +==============+========================================================+
+   | Cheyenne     | /glade/scratch/epicufsrt/containers                    |
+   +--------------+--------------------------------------------------------+
+   | Hera         | /scratch1/NCEPDEV/nems/role.epic/containers            |
+   +--------------+--------------------------------------------------------+
+   | Jet          | /mnt/lfs4/HFIP/hfv3gfs/role.epic/containers            |
+   +--------------+--------------------------------------------------------+
+   | Orion        | /work/noaa/epic-ps/role-epic-ps/containers             |
+   +--------------+--------------------------------------------------------+
+
+.. note::
+   Singularity is not available on Gaea, and therefore, container use is not supported on Gaea. 
+
+Users can simply copy the container to their local working directory. For example, on Jet:
+
+.. code-block:: console
+
+   cp /mnt/lfs4/HFIP/hfv3gfs/role.epic/containers/ubuntu20.04-intel-landda.img .
+
+Other Systems
+----------------
+
+On other systems, users can build the singularity container from a public Docker :term:`container` image. 
+
+.. code-block:: console
+
+   singularity build ubuntu20.04-intel-landda.img docker://noaaepic/ubuntu20.04-intel-landda:release-public-v1.0.0
+
```

Comment:
Alternatively, they can download the container from the s3 bucket, which might be faster, depending on download speed.